study/test/src/org/labkey/test/tests/study/StudySimpleExportTest.java

```diff
@@ -1070,14 +1071,13 @@ public void verifySpecimenSettings()
 
         waitAndClickAndWait(Locator.linkWithText("Manage Requestability rules"));
         log("Export Requestability Rules");
-        _extHelper.clickMenuButton(false, "Add Rule", "Custom Query");
-        _extHelper.waitForExtDialog("Add Custom Query Rule");
-        _extHelper.selectComboBoxItem("Schema:", "auditLog");
-        _extHelper.selectComboBoxItem("Query:", "FileSystem");
-        _extHelper.selectComboBoxItem("Mark vials:", "Unavailable");
+        clickButton("Add Rule", 0);
+        click(Locator.menuItem("Custom Query"));
+        _ext4Helper.selectComboBoxItem("Schema:", "auditLog");
+        _ext4Helper.selectComboBoxItem("Query:", "FileSystem");
+        _ext4Helper.selectComboBoxItem("Mark vials:", "Unavailable");
         clickButton("Submit", 0);

```

Comment:
Use helpers for menu and window to properly wait for page to change.
```suggestion
        _ext4Helper.clickExt4MenuButton(false, Ext4Helper.Locators.ext4Button("Add Rule"), false, "Custom Query");
        Window window = new Window("Add Custom Query Rule", getDriver()); // org.labkey.test.components.ext4.Window
        _ext4Helper.selectComboBoxItem("Schema:", "auditLog");
        _ext4Helper.selectComboBoxItem("Query:", "FileSystem");
        _ext4Helper.selectComboBoxItem("Mark vials:", "Unavailable");
        window.clickButton("Submit", true);
```